.eslintrc.cjs

```diff
@@ -1,22 +1,23 @@
 module.exports = {
-	root: true,
-	env: { browser: true, node: true, es2020: true },
-	extends: [
-		"eslint:recommended",
-		"plugin:react/recommended",
-		"plugin:react/jsx-runtime",
-		"plugin:react-hooks/recommended",
-	],
-	ignorePatterns: ["dist", ".eslintrc.cjs"],
-	parserOptions: { ecmaVersion: "latest", sourceType: "module" },
-	settings: { react: { version: "18.2" } },
-	plugins: ["react-refresh"],
-	rules: {
-		"react/jsx-no-target-blank": "off",
-		"react-refresh/only-export-components": [
-			"warn",
-			{ allowConstantExport: true },
-		],
-		"eslintreact/no-unescaped-entities": false,
-	},
+  root: true,
+  env: { browser: true, node: true, es2020: true },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:react/jsx-runtime',
+    'plugin:react-hooks/recommended',
+  ],
```

Comment:
I think we want to install the prettier plugin here to prevent eslint conflicts with prettier - chatgpt says do `npm install --save-dev eslint-config-prettier eslint-plugin-prettier prettier` and then add these 3 lines to the config:

```
  extends: [
    ...
    'plugin:prettier/recommended',
  ],
  plugins: ['react-refresh', 'prettier'], 
  rules: {
     ...
    'prettier/prettier': 'warn',
  },
```